src/components/NestedScrollView.js

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import {ScrollView, View} from 'react-native';
+import styles from '../styles/styles';
+
+const propTypes = {
+    /** Any children to display */
+    children: PropTypes.node,
+};
+
+const defaultProps = {
+    children: null,
+};
+
+const NestedScrollView = props => (
+    <ScrollView
+        horizontal
+        contentContainerStyle={[styles.flex1, styles.h100, styles.w100]}
+        scrollEnabled={false}
+    >
+        <View style={[styles.w100]}>
```

Comment:
Is this View really necessary?

Reply:
https://github.com/Expensify/App/issues/7547#issuecomment-1032851907

Reply:
First of all, H100 and w100is not required for `contentContainerStyle`. It should be managed by flex properties. Only add this if needed for this use.